modules/securityhub/variables.tf

```diff
@@ -37,6 +37,11 @@ variable "central_resource_collector_region" {
   type        = string
 }
 
+variable "organization_admin_account" {
+  description = "The name of the organization admin account"
+  type        = string
+}
+
```

Comment:
This should not be required. We could take it out, or leave it in so it could be overridden, but it should at least default to `account_map.outputs.root_account_account_name`.

Reply:
fixed